src/org/camunda/latera/kotlin/bss/connectors/JasperReport.kt

```diff
@@ -78,6 +78,23 @@ data class ExecutionStatusResponse(
  */
 class JasperReport(val url: String, val user: String, val password: String) {
   private val baseUrl: java.net.URI = java.net.URI(this.url)
+    fun createExecuteReportParams(params: LinkedHashMap<String, Any>): ExecuteReportParams {
```

Comment:
Looks like the wrong indent.

Also, I can't see any call of this function and any docs as well.